pkg/setting/setting.go

```diff
@@ -1248,6 +1249,7 @@ func readSecuritySettings(iniFile *ini.File, cfg *Cfg) error {
 	cfg.DisableInitAdminCreation = security.Key("disable_initial_admin_creation").MustBool(false)
 	cfg.AdminUser = valueAsString(security, "admin_user", "")
 	cfg.AdminPassword = valueAsString(security, "admin_password", "")
+	cfg.AdminEmail = valueAsString(security, "admin_email", "")
```

Comment:
I think using `cfg.AdminUser + "@localhost"` as the fallback/default value would be reasonable, what do you think @Jguer?

Reply:
:+1: I agree, it would also avoid breaking the previous default behavior

Reply:
I believe that `defaults.ini` will take precedence and set the email to `admin@localhost` even when `admin_user` has changed, so there'd be a ever so slight modification of current behavior.

Reply:
To be clear, I think that's fine and nothing to stress about, it's just good to be aware that that might happen to someone.